src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -12305,9 +12305,12 @@ uint32 Unit::GetModelForForm(ShapeshiftForm form, uint32 spellId) const
     return modelid;
 }
 
-void Unit::JumpTo(float speedXY, float speedZ, bool forward)
+void Unit::JumpTo(float speedXY, float speedZ, bool forward, Position const* dest)
```

Comment:
should we use Optional<Position> for this ?

Reply:
That would make for better semantics, yes.